q2_vsearch/tests/test_cluster_features.py

```diff
@@ -508,34 +438,191 @@ def test_no_overlapping_feature_ids(self):
         with self.assertRaisesRegex(ValueError,
                                     expected_regex='Some feat.*f1.*'):
             cluster_features_closed_reference(
-                sequences=input_sequences, table=input_table,
-                reference_sequences=ref_sequences, perc_identity=1.0)
+                sequences=self.input_sequences, table=input_table,
+                reference_sequences=self.ref_sequences_1, perc_identity=1.0)
 
+    def test_highest_count_clustering(self):
```

Comment:
Let's call this `test_1_percent_clustering_alt_abundances` and move it up so it's next to `test_1_percent_clustering`. 

Maybe the comment can note which specific feature is now most abundant, and how that changes the result.